### ISSUE TITLE
Let auth users create multiple registrations/profiles

### ISSUE THREAD

**Issue by madduck** [CTPUG/wafer#146]
A nice-to-have that came up in the wake of requiring SSO for DebConf for DC14/DC15 was that one person ought to be able to create multiple attendee profiles, e.g. a parent registering the entire family, or an HR person registering the entire dev team.

Doing so would require changing the current 1:1 UserProfile to a 1:n AttendeeProfile model, but that doesn't seem too invasive a change, especially if we opt to keep the current names for simpler migrations (though obviously it'd help to be more explicit in the naming).

However, everywhere in the code where Users are referenced (e.g. talks, pages), the references would need to be updated to AttendeeProfile.

This is a medium-priority item for DebConf and as such something I might well hack on soon. I am recording the issue here in case there are comments or there's feedback/opposition/support/etc.

**Comment by stefanor** [comment]
> Doing so would require changing the current 1:1 UserProfile to a 1:n AttendeeProfile model

Another way is to give someone the ability to trivially users themselves (and then they have some kind of delegate access to these users accounts).

It has to be simple if HR people are going to use it, though. i.e. just entering names and email addresses.

**Comment by madduck** [comment]
@stefanor good point actually. We do want additional data later (e.g. t-shirt size, doubtful HR has that), so individual auth access would be useful later on.

Possibly a simple solution would be to store a secret key with each profile, along with a URL that, if visited by a different auth user, simply reassociates the profile.

**Comment by madduck** [comment]
Just for reference, I think @stefanor and I came to the conclusion that the easiest way to do this would be to provide a sort of bulk creation form, allowing e.g. HR to create the users, but requiring each user themselves to activate their account.

For families, this will mean that parents need to use email addresses for their kids, but I suppose that's what `+`-extensions are for.
